runners/console/index.ts

```diff
@@ -655,7 +654,7 @@ export class Console extends Runner {
 
         let process = child_process.spawnSync(command, { shell: true, cwd: directory, input: input, maxBuffer: Infinity, env: this.env });
         if(process.status != 0) {
-            console.log("Error executing command: " + command + " (exit code: " + process.status + ")");
+            console.log("Error executing runCommand.command: " + command + " (exit code: " + process.status + ")");
```

Comment:
In strings you do not have to change it to runCommand. There are several other places in the code.